jsonHttpHandler/main.go

```diff
@@ -35,7 +36,7 @@ type Globals interface {
 
 type GlobalsReceivingHandlerFunc func(Globals) http.HandlerFunc
 
-type Middleware func(http.HandlerFunc) http.HandlerFunc
+type Middleware func(Globals, http.HandlerFunc) http.HandlerFunc
```

Comment:
Unlike the other types of `Middleware`s that I've seen, our `Middleware`s also accept the `Globals`.